tests/components/home_connect/test_init.py

```diff
@@ -192,41 +191,6 @@ async def test_token_refresh_success(
     )
 
 
-async def test_setup(
-    hass: HomeAssistant,
-    integration_setup: Callable[[], Awaitable[bool]],
-    config_entry: MockConfigEntry,
-    setup_credentials: None,
-) -> None:
-    """Test setting up the integration."""
-    assert config_entry.state == ConfigEntryState.NOT_LOADED
-
-    assert await integration_setup()
-    assert config_entry.state == ConfigEntryState.LOADED
-
-    assert await hass.config_entries.async_unload(config_entry.entry_id)
-    await hass.async_block_till_done()
-
-    assert config_entry.state == ConfigEntryState.NOT_LOADED
-
-
-async def test_update_throttle(
-    appliance: Mock,
-    hass: HomeAssistant,
-    config_entry: MockConfigEntry,
-    integration_setup: Callable[[], Awaitable[bool]],
-    setup_credentials: None,
-    platforms: list[Platform],
-    get_appliances: MagicMock,
-) -> None:
-    """Test to check Throttle functionality."""
-    assert config_entry.state == ConfigEntryState.NOT_LOADED
-
-    assert await integration_setup()
-    assert config_entry.state == ConfigEntryState.LOADED
-    assert get_appliances.call_count == 0
```

Comment:
Why are these being removed?

Reply:
`test_setup` is a duplicate of [test_api_setup](https://github.com/home-assistant/core/blob/6045c2bb087d057c90cbc999dddff0ce3e1392a2/tests/components/home_connect/test_init.py#L120-L137)
and
`test_update_throttle` I felt was not a good test since it depends on the order the tests are executed in. I noticed that when I tested on my local (`pytest -n 1`) fine and then saw failing tests in github when using multiple executors (`-n auto`). The line shows as covered in the report so I just removed it.